test/cli/test_detect.py

```diff
@@ -134,32 +134,6 @@ def test_single_directory_service_annotation(self):
                 + "service_annotation_v1.proto L40: An existing message `FooMetadata` is removed.\n",
             )
 
-    def test_mutiple_directories(self):
-        # Mock the stdout so that the unit test does not
-        # print anything to the console.
-        with patch("sys.stdout", new=StringIO()):
-            runner = CliRunner()
-            result = runner.invoke(
-                detect,
-                [
-                    "--original_api_definition_dirs=test/testdata/protos/enum/v1,test/testdata/protos/message/v1",
-                    "--update_api_definition_dirs=test/testdata/protos/enum/v1beta1,test/testdata/protos/message/v1beta1",
-                    "--original_proto_files=test/testdata/protos/enum/v1/enum_v1.proto,test/testdata/protos/message/v1/message_v1.proto",
-                    "--update_proto_files=test/testdata/protos/enum/v1beta1/enum_v1beta1.proto,test/testdata/protos/message/v1beta1/message_v1beta1.proto",
-                    "--human_readable_message",
-                ],
-            )
-            self.assertEqual(result.exit_code, 0)
-            self.assertEqual(
-                result.output,
-                "message_v1beta1.proto L7: Type of an existing field `id` is changed from `int32` to `string`.\n"
-                + "message_v1beta1.proto L8: Name of an existing field is changed from `email` to `email_address`.\n"
-                + "message_v1beta1.proto L21: Repeated state of an existing field `phones` is changed.\n"
-                + "message_v1beta1.proto L22: An existing field `single` is moved out of One-of.\n"
-                + "message_v1.proto L18: An existing field `type` is removed.\n"
-                + "enum_v1.proto L5: An existing Enum `BookType` is removed.\n",
-            )
-
```

Comment:
The multiple directory test is removed since 1) the oslogin already covers this case 2) the api definitions `message/` and `enums/`are not related or imported, so it is not a good test case.